src/app/signup/page.tsx

```diff
@@ -0,0 +1,15 @@
+import { Container, Typography, Box } from '@mui/material'
+import SignUpForm from '../../components/features/signup/SignUpForm'
+
+export default function SignUpPage() {
+  return (
+    <Container maxWidth="sm">
```

Comment:
We can have a layout.tsx file only for these.
https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#layout-pattern